SparkleSpin/AppDelegate.swift

```diff
@@ -44,3 +45,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
 }
 
+extension AppDelegate {
+    
+    func styleNavigationBar() {
+        let navigationBarAppearance = UINavigationBar.appearance()
+        let attributes = [
+            NSAttributedString.Key.foregroundColor: ThemeColor.Light.secondaryColor,
+            NSAttributedString.Key.font: Font.wheelBodyFont
+        ]
+        navigationBarAppearance.barTintColor = ThemeColor.Light.primaryColor
+        navigationBarAppearance.tintColor = ThemeColor.Light.accentColorOne
+        navigationBarAppearance.titleTextAttributes = attributes as [NSAttributedString.Key : Any]
+        navigationBarAppearance.shadowImage = UIImage()
+    }
+}
```

Comment:
I don't know if you have read [this article](https://basememara.com/protocol-oriented-themes-for-ios-apps/) Nothing wrong with your code and you don't have to implement what's in there, just reminded me the article and I wanted to share because I think is an interesting approach and I think it could be useful if not for now for the future.

Reply:
I just gave the article a read. Great resource! Definitely demonstrates robust theming, and I can see how that method would complement the way I'm handling styling like fonts and colors currently (values held in properties of specific structs). Will seriously consider implementing something like that in the future, thank you! 😄